<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="framecentral">
     <property name="maximumSize">
      <size>
       <width>771</width>
       <height>521</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>681</width>
        <height>371</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1600</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>Pacote de Viagens</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CbOrig">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 9px;
</string>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recife - Pernambuco</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fortaleza - Ceara</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S達o Paulo - S達o Paulo</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Origem:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Destino:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CbDest">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 9px;</string>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recife - Pernambuco</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fortaleza - Ceara</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S達o Paulo - S達o Paulo</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Dados do Cliente:</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>-10</y>
         <width>16</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Nome completo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>CPF:</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>-10</y>
         <width>16</width>
         <height>481</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>200</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Formas de Pagamento:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="caixaData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 214, 214);
border-radius: 9px;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Data da viagem:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CbPag">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 9px;
color: rgb(255, 255, 255);
</string>
       </property>
       <item>
        <property name="text">
         <string>Selecione</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cr辿dito</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D辿bito</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pix</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="CaixaCPF">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>140</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 218, 218);
background-color: rgb(255, 255, 255);
border-radius: 9px;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="caixaNome">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>70</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 218, 218);
background-color: rgb(255, 255, 255);
border-radius: 9px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtnReserva">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>340</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 214, 214);
border-radius: 9px;

QPushButton:hover(
	background-color: 
color: rgb(255, 170, 255);
	
)</string>
       </property>
       <property name="text">
        <string>Reservar</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
